owncloudTestUtil/src/main/java/com/owncloud/android/testutil/oauth/ClientRegistrationRequest.kt

```diff
@@ -22,7 +22,7 @@ import com.owncloud.android.domain.authentication.oauth.model.ClientRegistration
 import com.owncloud.android.testutil.OC_REDIRECT_URI
 
 val OC_CLIENT_REGISTRATION_REQUEST = ClientRegistrationRequest(
-    registrationEndpoint = OC_OIDC_SERVER_CONFIGURATION.registrationEndpoint,
+    registrationEndpoint = OC_OIDC_SERVER_CONFIGURATION.registrationEndpoint!!,
     clientName = "Android Client v2.17",
```

Comment:
Could this String not being hardcoded? Just asking, in order to to make it easier when we release new versions

Reply:
It is used for testing proposals, nothing relevant. This is in the owncloudTestUtil module. We could remove the `v2.17` but it won't affect at all.